patches/ava+5.3.1.patch

```diff
@@ -126,6 +141,21 @@ index b1989a4..fa1617f 100644
  			comment: evt.logs,
  			error: evt.err ? dumpError(evt.err) : null,
  			index: ++this.i,
+diff --git a/node_modules/ava/lib/run-status.js b/node_modules/ava/lib/run-status.js
+index 8471345..3110074 100644
+--- a/node_modules/ava/lib/run-status.js
++++ b/node_modules/ava/lib/run-status.js
+@@ -230,6 +230,10 @@ export default class RunStatus extends Emittery {
+ 			|| this.stats.uncaughtExceptions > 0
+ 			|| this.stats.unhandledRejections > 0
+ 		) {
++      const allowed = Number(process.env.AGORIC_AVA_ALLOW_UNHANDLED_REJECTIONS);
```

Comment:
I think I would prefer for this to be an exact expected value, like if it was a plan. 
```suggestion
+      const expectedUnhandledRejections = Number(process.env.AGORIC_AVA_EXPECTED_UNHANDLED_REJECTIONS);

```

Reply:
> exact expected value

That doesn't compose well with allowing developers to run a subset of the tests.

Reply:
Fair

Reply:
As per your preference, this patch is now an exact expected number of unhandled promise rejections.